### PR TITLE
Review UI text on the Department Management page.

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Signed-off-by: Patrick-LuoYu <patrickluo@yunify.com>
/kind documentation
/assign @leoendless @RolandMa1986 @Sherlock113

### CODE REVIEW

**Comment by Sherlock113** [comment]
Please solve the conflict first.

File: locales/en/group.js [hunk 1]
```diff
@@ -17,25 +17,33 @@
...
+  'User Group Assigned': 'Department',
 
   DELETE_PARENT_GROUP_TIP:
     'Are you sure you want to delete the sub-department <strong>{group_name}</strong>? When deleting the department, its sub-departments will also be deleted, and the authorization of all members will be cancelled.',
   DELETE_GROUP_TIP:
-    'Are you sure to delete the sub-department <strong>{group_name}</strong>? When deleting the department, the authorization of all members will also be cancelled.',
-}
+    'Are you sure you want to delete the department <strong>{group_name}</strong>? The associated roles will be unbound from the users.',
+  DELETE_PARENT_GROUP_TIP:
+    'Are you sure you want to delete the department <strong>{group_name}</strong>? Its sub-departments will also be deleted and the associated roles will be unbound from the users.',
```
Comment (Sherlock113) [inline locales/en/group.js:45]: Two identical keys (Line 40 and line 44). Delete a key.

File: locales/en/group.js [hunk 2]
```diff
@@ -17,25 +17,33 @@
...
+  GROUP_USER_TIP:
+    'The department contains <strong>{count}</strong> user. Please remove the user first.',
+  GROUP_USERS_TIP:
+    'The department contains <strong>{count}</strong> users. Please remove the users first.',
```
Comment (Sherlock113) [inline locales/en/group.js:49]: Where are these two keys?

File: locales/en/group.js [hunk 3]
```diff
@@ -17,25 +17,33 @@
...
+  'Add New Department': 'Create Department',
+  'Add New Subdepartment': 'Create Department',
   NO_DEPARTMENT_TIP:
-    'No available department at the moment. Please create one on the right.',
+    'No department available. Please create a department on the right.',
   'Workspace role': 'Workspace Role',
-  WORKSPACE_ROLE_DESC: 'Assign roles to the entire user group',
-  'Bind Project Role': 'Bind Project Role',
+  GROUP_WORKSPACE_ROLE_DESC:
+    'The workspace role will be assigned to all members in the department.',
```
Comment (Sherlock113) [inline locales/en/group.js:31]: `GROUP_WORKSPACE_ROLE_DESC` is a new key. Add this key in other language files as well.

File: locales/en/group.js [hunk 4]
```diff
@@ -17,25 +17,33 @@
  */
 
 module.exports = {
-  'Add the member to': 'Add the member to <strong>{group}</strong>',
-  'Department Name': 'Department Name',
+  'Add the member to':
+    'Are you sure you want to assign the user to <strong>{group}</strong>?',
+  'Department Name': 'Name',
+  'Department Alias': 'Alias',
```
[the remark this answers is not in the document]
Comment (Patrick-LuoYu) [reply locales/en/group.js:23]: Issues about the Chinese UI will be resolved in another PR.

File: locales/en/group.js (hunk 3 above)
Comment (Patrick-LuoYu) [reply locales/en/group.js:31]: Issues about the Chinese UI will be resolved in another PR.

File: locales/en/group.js (hunk 2 above)
Comment (Patrick-LuoYu) [reply locales/en/group.js:49]: The keys are reserved for future features.

Comment (RolandMa1986) [reply locales/en/group.js:49]: @xuliwenwenwen, please confirm whether those 2 keys are still in use, otherwise, we should remove them.

Comment (Patrick-LuoYu) [reply locales/en/group.js]: @RolandMa1986 The keys have been removed from the master branch. Therefore it should also be removed here.

**Pull request merged**